tests/prague/eip7685_general_purpose_el_requests/test_multi_type_requests.py

```diff
@@ -99,9 +102,15 @@ def single_consolidation_from_contract(i: int) -> ConsolidationRequestContract:
     return ConsolidationRequestContract(requests=[single_consolidation(i)])
 
 
-def get_permutations(n: int = 3) -> Generator[Any, None, None]:
+def get_permutations(
+    n: int = 3,
+) -> Generator[
+    DepositRequest | WithdrawalRequest | ConsolidationRequest,
+    None,
+    None,
+]:
```

Comment:
```suggestion
def get_permutations(n: int = 3) -> Generator[ParameterSet, None, None]:
```
I think this was my fault from last review, but this should be the way to go.